packages/rocketchat-emojione/lib/EmojiPicker.js

```diff
@@ -52,6 +52,9 @@ RocketChat.EmojiPicker = {
 		return this.recent;
 	},
 	setPosition() {
+		if (window.matchMedia('all and (min-width: 360px)').matches)
+			return $('.emoji-picker')
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'return'.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2562086674/issues/source?bid=3199634&fileBranchId=3293836#l56)